.ci/update-branch-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Given the current and next branches, in major.minor format, this script will update
+# those versions.
+#
+# Parameters:
+#	$1 -> the version to be searched. Mandatory.
+#	$2 -> the version to be updated. Mandatory.
+#	$3 -> whether to create a branch where to commit the changes to.
+#		  this is required when reusing an existing Pull Request.
+#		  Optional. Default true.
+#
+set -euo pipefail
+MSG="parameter missing."
+CURRENT=${1:?$MSG}
+NEXT=${2:?$MSG}
+CREATE_BRANCH=${3:-true}
+
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+
+if [ "${OS}" == "darwin" ] ; then
+	SED="sed -i .bck"
+else
+	SED="sed -i"
+fi
+
```

Comment:
Would this place a good one to bump the upgrade agent version?

Reply:
What file did I miss?